atom/browser/common_web_contents_delegate.cc

```diff
@@ -522,15 +522,15 @@ void CommonWebContentsDelegate::DevToolsIndexPath(
       scoped_refptr<DevToolsFileSystemIndexer::FileSystemIndexingJob>(
           devtools_file_system_indexer_->IndexPath(
               file_system_path, excluded_folders,
-              base::Bind(
+              base::BindRepeating(
```

Comment:
All of these devtools callbacks can be `once` but we can't change it because of the api signature in `DevTooslFileSystemIndexer`, thats sad.

Reply:
yeah :(